docs/REFERENCE.md

```diff
@@ -3,141 +3,143 @@
-  * [`.chezmoiignore`](#chezmoiignore)
-  * [`.chezmoiremove`](#chezmoiremove)
-  * [`.chezmoitemplates`](#chezmoitemplates)
-  * [`.chezmoiversion`](#chezmoiversion)
-* [Commands](#commands)
-  * [`add` *targets*](#add-targets)
-  * [`apply` [*targets*]](#apply-targets)
-  * [`archive`](#archive)
-  * [`cat` *targets*](#cat-targets)
-  * [`cd`](#cd)
-  * [`chattr` *attributes* *targets*](#chattr-attributes-targets)
-  * [`completion` *shell*](#completion-shell)
-  * [`data`](#data)
-  * [`diff` [*targets*]](#diff-targets)
-  * [`docs` [*regexp*]](#docs-regexp)
-  * [`doctor`](#doctor)
-  * [`dump` [*targets*]](#dump-targets)
-  * [`edit` [*targets*]](#edit-targets)
-  * [`edit-config`](#edit-config)
-  * [`execute-template` [*templates*]](#execute-template-templates)
-  * [`forget` *targets*](#forget-targets)
-  * [`git` [*arguments*]](#git-arguments)
-  * [`help` *command*](#help-command)
-  * [`hg` [*arguments*]](#hg-arguments)
-  * [`init` [*repo*]](#init-repo)
-  * [`import` *filename*](#import-filename)
-  * [`manage` *targets*](#manage-targets)
-  * [`managed`](#managed)
-  * [`merge` *targets*](#merge-targets)
-  * [`purge`](#purge)
-  * [`remove` *targets*](#remove-targets)
-  * [`rm` *targets*](#rm-targets)
-  * [`secret`](#secret)
-  * [`source` [*args*]](#source-args)
-  * [`source-path` [*targets*]](#source-path-targets)
-  * [`unmanage` *targets*](#unmanage-targets)
-  * [`unmanaged`](#unmanaged)
-  * [`update`](#update)
-  * [`upgrade`](#upgrade)
-  * [`verify` [*targets*]](#verify-targets)
-* [Editor configuration](#editor-configuration)
-* [Umask configuration](#umask-configuration)
-* [Template execution](#template-execution)
-* [Template variables](#template-variables)
-* [Template functions](#template-functions)
-  * [`bitwarden` [*args*]](#bitwarden-args)
-  * [`bitwardenFields` [*args*]](#bitwardenfields-args)
-  * [`gopass` *gopass-name*](#gopass-gopass-name)
-  * [`include` *filename*](#include-filename)
-  * [`ioreg`](#ioreg)
-  * [`joinPath` *elements*](#joinpath-elements)
-  * [`keepassxc` *entry*](#keepassxc-entry)
-  * [`keepassxcAttribute` *entry* *attribute*](#keepassxcattribute-entry-attribute)
-  * [`keyring` *service* *user*](#keyring-service-user)
-  * [`lastpass` *id*](#lastpass-id)
-  * [`lastpassRaw` *id*](#lastpassraw-id)
-  * [`lookPath` *file*](#lookpath-file)
-  * [`onepassword` *uuid* [*vault-uuid*]](#onepassword-uuid-vault-uuid)
-  * [`onepasswordDocument` *uuid* [*vault-uuid*]](#onepassworddocument-uuid-vault-uuid)
-  * [`onepasswordDetailsFields` *uuid* [*vault-uuid*]](#onepassworddetailsfields-uuid-vault-uuid)
-  * [`pass` *pass-name*](#pass-pass-name)
-  * [`promptBool` *prompt*](#promptbool-prompt)
-  * [`promptInt` *prompt*](#promptint-prompt)
-  * [`promptString` *prompt*](#promptstring-prompt)
-  * [`secret` [*args*]](#secret-args)
-  * [`secretJSON` [*args*]](#secretjson-args)
-  * [`stat` *name*](#stat-name)
-  * [`vault` *key*](#vault-key)
+
```

Comment:
It looks like something reformatted the entire `REFERENCE.md` file. Could you update the commit to only include the relevant changes?

Reply:
Sure, I found it odd too, but I thought that was the normal behaviour. Will update accordingly.

Reply:
Auch, that was my editor ;')